cmd/nerdctl/login.go

```diff
@@ -82,6 +86,7 @@ func processLoginOptions(cmd *cobra.Command) (types.LoginCommandOptions, error)
 		password = strings.TrimSuffix(string(contents), "\n")
 		password = strings.TrimSuffix(password, "\r")
 	}
+
```

Comment:
Unneeded

Reply:
Funny I am not the only one doing this :-).
I do personally like that style and feel it does make things more readable by having return statements stand up more.

Reply:
@Shikachuu would you like to remove this extraneous line so that we can merge this?

Thanks again!